templates/custom/index.html

```diff
@@ -32,10 +45,38 @@ <h1>Create custom tests on your fork</h1>
         <div class="medium-6 columns">
             {{ macros.render_field(addTestFork.platform) }}
         </div>
+        <div class="medium-12 columns">
+                <label for="regression_test">Choose Regression Tests</label>
+                {% for category in categories %}
+                <h6 class="category-header" data-category="{{ category.id }}">{{ category.name }} </h6>
+                <div class="hide" data-category="{{ category.id }}">
+                <input type="checkbox" name="{{ category.name }}" onClick="toggle(this)" /> Select All<br/>
```

Comment:
Try to bind this at runtime with JQuery instead of hardcoded. Slap a class selector on these checkboxes, and then do a `$('.selector').on('click', toggle);`